plugin/pkg/admission/storageclass/default/admission.go

```diff
@@ -88,7 +88,7 @@ func (a *claimDefaulterPlugin) SetInternalClientSet(client internalclientset.Int
 	}
 }
 
-// Validate ensures an authorizer is set.
+// Validate ensures clientset and store are set.
```

Comment:
so already this comment appears out of date since it validates that there is a client, a reflector, an a store.

does this godoc provide any value at all for a private type?

can we combine a change like this w/ something that also provides a more meaningful enhancement to say the testing of the feature?